### PR TITLE
translated file 404.md

### PR DESCRIPTION
**Pull request by spielers**
please merge this

### CODE REVIEW

**Comment by arshadkazmi42** [comment]
@spielers I am impressed by your active work here. But since we don't have active reviewers. I would request you to hold on, on sending new PRs.
There are already two open PRs.
Lets get all those closed before sending new ones

Thanks

**Comment by spielers** [comment]
ok please review the open PR and let me know the errors and please review this pr and let me a part of this organization

**Comment by arshadkazmi42** [comment]
> @spielers I am impressed by your active work here. But since we don't have active reviewers. I would request you to hold on, on sending new PRs.
> There are already two open PRs.
> Lets get all those closed before sending new ones
> 
> Thanks

Hey. sorry about this.
I got confused with Gatsby and ReactJs.
This PR is fine. Thank you for this. I will get to it soon 👍

**Comment by spielers** [comment]
thanks u, it's just 2 lines of code please merge this.

**Comment by arshadkazmi42** [comment]
@spielers we will get to it. We have alot of open PRs pending for review.
Hope you understand

**Comment by spielers** [comment]
@arshadkazmi42 thank you

File: content/404.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
 layout: single
-title: Page Not Found
+title: पेज नहीं मिला
 permalink: 404.html
 ---
 
-We couldn't find what you were looking for.
-
-Please contact the owner of the site that linked you to the original URL and let them know their link is broken.
+जिसकी आपको तलाश थी,हमें वह नहीं मिला।
```
Comment (arshadkazmi42) [inline content/404.md]: `जिसकी आपको तलाश थी,हमें वह नहीं मिला।` => `जिसकी आपको तलाश थी, हमें वह नहीं मिला।`

minor spacing issue

File: content/404.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
+कृपया उस साइट के ओनर से संपर्क करें जिसने आपको मूल URL से जोड़ा है और उन्हें बताएं कि उनका लिंक टूट गया है।
```
Comment (arshadkazmi42) [inline content/404.md]: add line break at line 8

**Comment by spielers** [comment]
@arshadkazmi42  sir, I want to be a part of this react community so please merge my PR, else add me in maintainers list please, it's really important for me.

**Comment by arshadkazmi42** [comment]
> @arshadkazmi42 sir, I want to be a part of this react community so please merge my PR, else add me in maintainers list please, it's really important for me.

There are a lot of pages pending in #1 
You can pick one page from there to do contribution to this repo.
For maintainers, we are not looking for any new maintainers right now.

File: content/404.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-title: Page Not Found
+title: पेज नहीं मिला
 permalink: 404.html
 ---
 
-We couldn't find what you were looking for.
+जिसकी आपको तलाश थी, हमें वह नहीं मिला।
 
-Please contact the owner of the site that linked you to the original URL and let them know their link is broken.
+कृपया उस साइट के ओनर से संपर्क करें जिसने आपको मूल URL से जोड़ा है और उन्हें बताएं कि उनका लिंक टूट गया है।
```
Comment (saranshkataria) [inline content/404.md:9]: टूट गया है should be toota hua hai

Comment (spielers) [reply content/404.md]: fixed

**Comment by spielers** [comment]
changes fixed please merge this pr, I will start working on a new page when this gets merged.

**Comment by saranshkataria** [comment]
@spielers done. And for the next PR, please follow the process.

**Pull request merged**